ledger/src/blockstore.rs

```diff
@@ -1667,6 +1667,9 @@ impl Blockstore {
                 metrics,
             );
         }
+
+        self.synchronize_slot_meta_with_parent_meta(&mut shred_insertion_tracker);
```

Comment:
any reason not to wrap this alongside `maybe_update_parent_meta()` in a single function instead?  That function is already handling making the `parent_slot_meta` consistent: https://github.com/ksn6/alpenglow/blob/59a53d6cf818f6cc774713668f12bef4925e4e49/ledger/src/blockstore.rs#L2496-L2500

Would be logically nice to have all the weeds of the parent update stuff in one place